river/proba/gaussian.py

```diff
@@ -90,3 +95,203 @@ def sample(self):
+
+    @property
+    def mu(self):
+        """The mean value of the distribution."""
+        return list(
+            {
+                key1: values.mean.get()
+                for (key1, key2), values in self._var.matrix.items()
+                if key1 == key2
+            }.values()
+        )
+
+    @property
+    def var(self):
+        """The variance of the distribution."""
+        variables = sorted(list({var for cov in self._var.matrix.keys() for var in cov}))
+        # Initialize the covariance matrix array
+        cov_array = np.zeros((len(variables), len(variables)))
+
+        # Fill in the covariance matrix array
+        for i in range(len(variables)):
+            for j in range(i, len(variables)):
+                if i == j:
+                    # Fill in the diagonal with variances
+                    cov_array[i, j] = self._var[(variables[i], variables[j])].get()
+                else:
+                    # Fill in the off-diagonal with covariances
+                    cov_array[i, j] = self._var[(variables[i], variables[j])].get()
+                    cov_array[j, i] = self._var[(variables[i], variables[j])].get()
+        return cov_array
+
+    @property
+    def sigma(self):
+        """The standard deviation of the distribution."""
+        cov_array = self.var
+        return [[x**0.5 if x > 0 else float("nan") for x in row] for row in cov_array]
+
+    def __repr__(self):
+        mu_str = ", ".join(f"{m:.3f}" for m in self.mu)
+        var_str = "\n ".join("[" + " ".join(f"{s:.3f}" for s in row) + "]" for row in self.var)
+        return f"𝒩(μ=({mu_str}),\nσ^2=({var_str}))"
+
+    def update(self, x, w=1.0):
+        if w != 1.0:
+            warnings.warn("Weights not implemented yet.", RuntimeWarning)
+        self._var.update(x)
+        return self
+
+    def revert(self, x, w=1.0):
+        if w != 1.0:
+            # TODO: find out why not called during TimeRolling usage test
+            warnings.warn("Weights not implemented yet.", RuntimeWarning)  # pragma: no cover
+        self._var.revert(x)
+        return self
+
+    def __call__(self, x):
+        x = list(x.values())
+        var = self.var
+        if var is not None:
+            try:
+                return multivariate_normal(self.mu, var).pdf(x)
+            # TODO: validate occurence of ValueError
+            # The input matrix must be symmetric positive semidefinite.
+            except ValueError:  # pragma: no cover
+                return 0.0
+            # TODO: validate occurence of OverflowError
+            except OverflowError:  # pragma: no cover
+                return 0.0
+        return 0.0  # pragma: no cover
+
+    def cdf(self, x):
+        x = list(x.values())
+        return multivariate_normal(self.mu, self.var, allow_singular=True).cdf(x)
+
+    def sample(self):
+        return multivariate_normal(
+            self.mu,
+            self.var,
+        ).rvs()
```

Comment:
```suggestion
        return float(multivariate_normal(
            self.mu,
            self.var,
        ).rvs())
```

It's more "River-like" to return a `float`, and not a `np.float64`